frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -704,7 +704,7 @@
  "icon": "fa fa-cog",
  "issingle": 1,
  "links": [],
- "modified": "2025-04-15 14:12:11.443610",
+ "modified": "2025-05-16 22:10:27.590981",
```

Comment:
```suggestion
 "modified": "2025-04-15 14:12:11.443610",
```

When reverting changes, revert them cleanly to ensure good `git blame` in future. 

`git revert` or `git rebase -i` are your friends.